doc/site/en/guides/border-router/beaglebone-black.md

```diff
@@ -1,338 +1,204 @@
-size of the SD card. The [Expand the SD card
-image](#expand-the-sd-card-image) procedure uses `+7G` as the new partition
-size for an 8 GB SD card. This produces an image that is small enough to safely
-fit on any comparable 8 GB SD card (regardless of bad sectors) while still being
-large enough to build and install OTBR.
+The `BBONE-GATEWAY-CAPE` is not recognized by the BeagleBone by default because
+of a pin conflict. Add the configuration manually by editing the `uEnv.txt`
+with the command `sudo vim /boot/uEnv.txt` and make sure the following lines
+match:
 
-Use the ending sector of the data partition to calculate the entire byte size of
-the "data image" and truncate the IMG file at that byte offset. The simplest
-method is to use the `truncate` command. The `truncate` command is a standard
-Unix command line tool, and it is also present in the MS-Windows Git Bash
-distribution of MSYS.
+```
+###Custom Cape
+dtb_overlay=/lib/firmware/BB-GATEWAY-WL1837-00A0.dtbo
+###
+###Disable auto loading of virtual capes (emmc/video/wireless/adc)
+disable_uboot_overlay_emmc=1
+disable_uboot_overlay_video=1
+disable_uboot_overlay_audio=1
+disable_uboot_overlay_wireless=1
+disable_uboot_overlay_adc=1
+```
 
-As `root`, run `fdisk` on the device (SD card):
+The BeagleBone wilink setup scripts try to use connman by default to enable
+Wi-Fi AP activity. Edit the default configuration folder with the command `sudo
+vim /etc/default/bb-wl18xx` and make sure the variables match below:
 
 ```
-root@beaglebone:/home/debian# fdisk /dev/mmcblk0
-
-Welcome to fdisk (util-linux 2.25.2).
-Changes will remain in memory only, until you decide to write them.
-Be careful before using the write command.
+TETHER_ENABLED=no
+USE_CONNMAN_TETHER=no
 ```
 
-Print the current partition table to find the ending sector. In this
-example, the ending sector is `14688255`:
+Restart to make sure Network Manager can see the new interface.
 
 ```
-Command (m for help): p
-Disk /dev/mmcblk0: 7.2 GiB, 7744782336 bytes, 15126528 sectors
-Units: sectors of 1 * 512 = 512 bytes
-Sector size (logical/physical): 512 bytes / 512 bytes
-I/O size (minimum/optimal): 512 bytes / 512 bytes
-Disklabel type: dos
-Disk identifier: 0xca52207f
-
-Device         Boot Start      End  Sectors Size Id Type
-/dev/mmcblk0p1       8192 14688255 14680064   7G 83 Linux
+sudo shutdown -r now
 ```
 
-Quit `fdisk` and calculate the total size of the image:
+Once logged back in you can run `ifconfig` or `nmcli` to see the new `wlan`
+interface.
 
-1.  The last partition ends at sector `14688255`.
-1.  Each sector is 512 bytes.
-1.  The starting sector of an SD card is always `0`.  Add 1 byte to account
-    for this sector.
-1.  The total size is: `(14688255 + 1) * 512 = 7520387072`
+Warning: The startups scripts may take a few moments to enable the `wlan0`
+interface. If you do not see the interface, check `journalctl` to see if the
+system is having difficulty bringing up the interface.
```

Comment:
Callouts should be blockquote with > preceding the first line and the others indented

> Warning: